html/elements/meta.json

```
@@ -124,6 +124,469 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "height": {
+            "__compat": {
+              "description": "content=\"height=[value]\"",
+              "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Viewport_meta_tag#height",
+              "spec_url": "https://drafts.csswg.org/css-viewport/#viewport-meta",
```

Either omit the spec_url (if it's not specific to this feature), or use a Text Fragment:

```suggestion
              "spec_url": "https://drafts.csswg.org/css-viewport/#:~:text=width-,height,-initial-scale",
```

---

I’m not sure that linking to a specific list item fragment is more useful (i.e., it gives more info) than linking to a dedicated spec section. We will have to update links when the spec matures either way.

---

In the BCD call on October 29, we decided to use text fragments where anchors aren't available. Although this was in the context of behavioral subfeatures, I see no reason not to apply the same here.

I don't think it makes sense to repeat the parent `spec_url`.

---

I updated the spec URLs to link to IDs for specific properties or, otherwise, use fragment identifiers.

---

Thank you!